rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -1147,6 +1015,19 @@ public Void apply(NodeSource nodeSource) {
         }, preemptive, isTriggeredFromShutdownHook));
     }
 
+    public void addEligibleNode(RMNode rmnode) {
```

Comment:
is this really a public method ?

Reply:
it needs to be for the recovery manager

Reply:
I changed it into `recoverEligibleNodes(List<RMNode> eligibleNodes)` to make its usage more clear